pom.xml.versionsBackup

```diff
@@ -0,0 +1,366 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Do you need this? The regular pom file does not contain versions plugin, is this for later use?

Reply:
oh, I committed it on accident. Now I added it to the .gitignore